### DISCUSSION TITLE
How to bootstrap sync?

### DISCUSSION THREAD

**Discussion by levonk** [sboesen/remotely-sync#137] [Q&A] [answered]
This is a combination question & request.

I've been working on my desktop exclusively, I set up Remotely-Sync with OneDrive haven't even sync'd the first time yet, but I was confused on how to now sync with my android device?

I installed Obsidian, but before I can do anything, I have to decide whether I want a new vault or open an existing vault. Even after my data syncs to onedrive, I don't have one drive on my device yet (I was hoping I wouldn't need it, as remotely-sync would just connect to the cloud service).   

- Do I have to get my obsidian directory on the device to bootstrap first?
- Do I *need* to install OneDrive on the android device? Beyond getting it bootstrapped if that's how I choose to get the files there?
- Can I just create a new vault and then install remotely sync and sync to same one drive and have it take care of moving the files there?

Anyway, the request is, can the solution be added to the readme.md on the root of the repo?

Thanks for the help in advance!

**Comment by kadisonm** [comment, marked as answer]
Hi thanks for the question,

I agree the documentation isn't great for set up.

Part 1
1. On your main device, first make a local backup somewhere else on your computer.
2. Install the plugin to your obsidian vault.
3. Under 'Choose a remote service' select OneDrive
4. Press 'auth' and sign into OneDrive and follow the prompts.
5. Run a manual sync via the command or ribbon button (buttons on the left of vault)

Part 2
1. On your android device, create a new Obsidian vault called the exact same name as the vault on your computer. If your computer's vault name is "My Vault", it must also be called "My Vault" on your phone.
2. Enable community plugins and download Remotely Sync.
3. Under 'Choose a remote service' select OneDrive.
4. Press 'auth' and do the steps again.
5. Run a manual sync, this time the plugin will download the files previously uploaded to OneDrive.

Let me know if you have any other questions.
